frontend/src/pages/homepage/components/truck-list/truck-list.tsx

```diff
@@ -0,0 +1,58 @@
+import { Spinner } from '~/libs/components/components.js';
+import { TowTruckCard } from '~/libs/components/tow-truck-card/tow-truck-card.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useAppSelector, useMemo } from '~/libs/hooks/hooks.js';
+import { selectTrucks } from '~/slices/trucks/selectors.js';
+
+import styles from './styles.module.scss';
+
+type MockDataRating = {
+  averageRating: number;
+  reviewCount: number;
+};
+
+const getMockRating = (): MockDataRating => {
+  const averageRating = Math.floor(Math.random() * 3) + 3;
+  const reviewCount = Math.floor(Math.random() * 15) + 1;
+
+  return {
+    averageRating,
+    reviewCount,
+  };
+};
```

Comment:
Im not sure that ratings actually will be implemented, so you can remove this block, and when we will decide to add, then we will have a separate task for it